biogeochem/EDPhysiologyMod.F90

```diff
@@ -2553,7 +2555,7 @@ subroutine recruitment(currentSite, currentPatch, bc_in)
 
             ! but if the plant is seasonally (cold) deciduous, and the site status is flagged
             ! as "cold", then set the cohort's status to leaves_off, and remember the leaf biomass
-            if ((prt_params%season_decid(ft) == itrue) .and.                   &
+            if ((prt_params%phen_leaf_habit(ft) == ihard_season_decid) .and.                   &
```

Comment:
It looks like we could move this into the `select case` statement down on line 2571 correct?  This would mirror the work you made earlier in this file.

Reply:
Thanks for the suggestion! I moved this part to the `select case` block below.